ui/views_test.py

```diff
@@ -297,13 +304,17 @@ def test_collection_viewset_create(logged_in_apiclient):
     other_user = UserFactory()
     input_data = {
         'owner': other_user.id,
-        'title': 'foo title'
+        'title': 'foo title',
+        'view_lists': [],
+        'admin_lists': []
     }
     assert client.post(url, input_data, format='json').status_code == status.HTTP_403_FORBIDDEN
 
     # or if does not specify the owner id
     input_data = {
-        'title': 'foo title'
+        'title': 'foo title',
+        'view_lists': [],
+        'admin_lists': []
```

Comment:
Since it wasn't added as part of the PR I'd prefer to leave this test for now and wait for input from Giovanni.  I think this is to just make sure that each condition in the CollectionViewSet's if statement is being tested:
`if not request.user.is_superuser and (owner_id is None or int(owner_id) != request.user.id)`